### ISSUE TITLE
joi is now @hapi/joi

### ISSUE THREAD

**Issue by vincent-ogury** [evanshortiss/express-joi-validation#14]
This module doesn't enforce Joi dependancy. I have upgraded to @hapi/joi, and if I don't install the old Joi version, it doesn't work.

Can you provide a way to pass joi module to your library instance of requiring it?
Or can you move forward to @hapi/joi?

Thanks.

**Comment by evanshortiss** [comment]
@vincent-ogury you can pass Joi, but it seems I didn't mention it in docs. See [this code](https://github.com/evanshortiss/express-joi-validation/blob/master/express-joi-validation.js#L65).

Here's an example:

```js
const validator = require('express-joi-validation')({
  joi: require('@hapi/joi')
})
```

I will create a new release of this module with `@hapi/joi` as a `peerDependency`. The workaround above should be ok until I perform that release.

I also was shown the [celebrate](https://github.com/arb/celebrate) module recently. It seems to be sponsored so has better support though is slightly different to this one in some areas.

**Comment by vincent-ogury** [comment]
nice thx. I will take a look at celebrate also.

**Comment by evanshortiss** [comment]
@vincent-ogury released this update as 1.0.0 👍
